SpywareFilter/sections/cname_trackers.txt

```diff
@@ -62,6 +65,9 @@
 ! Lead Forensics : ghochv3eng.trafficmanager.net disguised trackers
 ! not included
 !
+! NameSilo : ab1n.net disguised trackers
+!#include https://raw.githubusercontent.com/AdguardTeam/cname-trackers/master/trackers/namesilo.txt
```

Comment:
It looks like a list of ad servers, so maybe it would be better to add it to Base filter?